Debug all the way!

ChielBruin opened the pull request:
- Added more nodes:
  - `battery_display`: a node that shows the battery charge
  - `ultrasonic_display`: a node that shows the distances measured by the ultrasonic sensors
- Improved `keyboard_sensors`
- Improved package structure
- Added 2 launch profiles:
  - `display.launch`: launches all display nodes
  - `turtle.launch`: launches the turtle and the turtleBridge

ChielBruin committed:
Added a node to view the battery charge

ChielBruin committed:
Added a node that displays the distance measurements

ChielBruin committed:
Minor fixes

Fixed some comments and the battery pecentage display

ChielBruin committed:
Refactored and improved keyboard_sensors

Better code layout and removed automatic resets of values

ChielBruin committed:
Renamed overview nodes to display nodes

ChielBruin committed:
Moved debug nodes to separate folders

ChielBruin committed:
Fixed cmake after renaming the file

ChielBruin committed:
Fixed a type in the filename

ChielBruin committed:
Added a launch profile for running the display nodes

ChielBruin committed:
Added launch profile for the turtle

This profile launches the turtleBridge and the turtle

ChielBruin committed:
Merge branch 'master' into sensor_overview

ChielBruin committed:
Merge branch 'master' into sensor_overview

Conflicts:
	catkin_ws/src/acr_debug/CMakeLists.txt

ChielBruin committed:
Moved WifiControls

marcelkuipers commented:
maybe it is nicer to reduce battery charge with a discrete amount, say 5 or 10 percent each time. Right now you have to press a lot of times before the charge gets low. Also, the charge will never become zero, do we want this?

marcelkuipers commented:
for the rest it looks good

ChielBruin committed:
Improved battery levels and ranges

Improved the display of the battery charge and ultrasonic range.
Also changed them from changing by percentage to cheanging by a constant value

ChielBruin commented:
Good point! I changed it

marcelkuipers commented on catkin_ws/src/acr_debug/src/keyboard_sensors/keyboard_sensors.py:
```diff
@@ -95,80 +124,60 @@ def getKey():
-		ult_max = 10000
-		ult_dist = 1999
+		ult_max = 500
+		ult_dist = 100
```
You should also update the min distance attribute in sensordata.hpp. This attribute is set to 2000, causing the SensorData class to always think there is something right in front of the robot.

ChielBruin committed:
Set min_dist to 50 cm

marcelkuipers commented:
okay, it looks good now
